web/src/components/build-feed/BuildFeedItem.vue

```diff
@@ -7,7 +7,7 @@
       <div class="flex flex-col mt-2">
         <div class="flex space-x-2 items-center">
           <Icon name="since" />
-          <span>{{ since }}</span>
+          <span v-tooltip="'Started at ' + started">{{ since }}</span>
```

Comment:
Could you try to mount the directives locally just for this component. (Next to `setup()`).

Reply:
`v-tooltip` is in three files (`web/src/components/build-feed/BuildFeedItem.vue`, `web/src/components/repo/build/BuildItem.vue`, and `web/src/views/repo/build/BuildWrapper.vue`) so I'd prefer to not have it specialized to those files and instead use a global import. Let me know if I'm misinterpreting what you're saying or if you want it changed regardless.

Reply:
Okay, we could to try to shrink it with `app.directive('tooltip', FloatingVue.VTooltip)` instead of `app.use(FloatingVue)`, but keep it globally.

Reply:
If CI passes, then it's all good!

Reply:
CI passed on my Woodpecker instance. Resolving.